### ISSUE TITLE
Anchor points in event calls

### ISSUE THREAD

**Issue by kimaldis** [cmiz/jquery.iocurve#1]
Is there a way to do this? Currently I'm using the .anchor points in the html but. obviously that's not the best way to do it.

Excellent plugin, btw.

**Comment by cmiz** [comment]
@kimaldis 
Thanks for use.
Now there is no event or method to get the anchor elements.
So I'm going to support a new event `anchors`.
```
$curve.on('anchors', function(ev, $anchors) {
    // $anchors is a jQuery object of all anchor elements.
    $anchors.each(function(){ ... });
});
```
This event will be callback when anchor elements are created or modified.
The argument `$anchors` is a jQuery object of all anchor elements.

Would this be useful?

**Comment by kimaldis** [comment]
Incredibly useful. Thanks.

**Comment by cmiz** [comment]
@kimaldis 
I've released version 0.0.6, which supports the new event `anchors`. Please try it.
https://cmiz.github.io/jquery.iocurve/#events
